app/components/work_packages/activities_tab/journals/item_component/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<%=
+  component_wrapper do
+    if journal.notes.present? || (journal.details.any? && !journal.initial?)
+      flex_layout do |journal_container|
+        if journal.notes.present?
```

Comment:
I don't get this nesting of statements. If you have `journal.notes.present?` in the inner, you don't need the other conditions?

Reply:
Done